vespa-http-client/src/main/java/com/yahoo/vespa/http/client/Result.java

```diff
@@ -191,12 +191,12 @@ public String toString() {
             b.append("Detail ");
             b.append("resultType=").append(resultType);
             if (exception != null) {
-                b.append("exception='").append(Exceptions.toMessageString(exception)).append("' ");
+                b.append(" exception='").append(Exceptions.toMessageString(exception)).append("' ");
```

Comment:
Should this be `(...).append('\'');` to avoid double spacing (a sign of the unrefined gentleman)?

Reply:
Sigh, yes. Fixing